# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/painting/RetainedDisplayListBuilder.h
# Commit: a7a2744670a7
# Full Hash: a7a2744670a7ce15ccafd8c59b909d90b8e2837a
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2024-01-11 09:52:49
# Regressor Bug: 1860328
# File Overlap Count: 2
# Description:
#   Bug 1860328 - Fix a bug where caret misses when dragging and dropping r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192377
# ==============================================================================

diff -r 62230fd018f6 -r a7a2744670a7 layout/painting/RetainedDisplayListBuilder.h
--- a/layout/painting/RetainedDisplayListBuilder.h	Wed Jan 10 16:50:49 2024 +0000
+++ b/layout/painting/RetainedDisplayListBuilder.h	Wed Jan 10 17:03:38 2024 +0000
@@ -272,7 +272,6 @@
 
   nsDisplayListBuilder mBuilder;
   RetainedDisplayList mList;
-  WeakFrame mPreviousCaret;
   RetainedDisplayListMetrics mMetrics;
   RetainedDisplayListData mData;
 };